node/src/test_utils/mod.rs

```diff
@@ -497,6 +487,32 @@ pub fn assert_eq_debug<T: Debug>(a: T, b: T) {
     assert_eq!(a_str, b_str);
 }
 
+#[cfg(test)]
+pub mod pure_test_only_utils {
+    use crate::apps::app_node;
+    use crate::test_utils::persistent_configuration_mock::PersistentConfigurationMock;
+    use masq_lib::multi_config::MultiConfig;
+    use std::mem::swap;
+    use std::ops::DerefMut;
+
+    pub fn make_simplified_multi_config(args: &[String]) -> MultiConfig {
```

Comment:
Should we convert from ArgsBuilder to &[String] entirely?

Reply:
I don't think so, but I don't think we won't make use of my utility either. I improved it and its very fast to use with &str parameters now. I can see a decent threshold between them two and also usefulness that holds for both in a different aspects.